cfgmgr/buffer_headroom_mellanox.lua

```diff
@@ -35,6 +35,26 @@ local state_db = "6"
 
 local ret = {}
 
+-- pause quanta should be taken for each operating speed is defined in IEEE 802.3 31B.3.7
+-- the key of table pause_quanta_per_speed is operating speed at Gb/s
+-- the value of table pause_quanta_per_speed is the number of pause_quanta
+local pause_quanta_per_speed = {}
+pause_quanta_per_speed[400000] = 905
+pause_quanta_per_speed[200000] = 453
+pause_quanta_per_speed[100000] = 394
+pause_quanta_per_speed[50000] = 147
+pause_quanta_per_speed[40000] = 118
+pause_quanta_per_speed[25000] = 80
+pause_quanta_per_speed[10000] = 67
+pause_quanta_per_speed[1000] = 2
+pause_quanta_per_speed[100] = 1
+
+-- if failed to get an pause_quanta from the table, then take a default one.
+local pause_quanta = pause_quanta_per_speed[port_speed]
+if pause_quanta == nil then
+    pause_quanta = 60
```

Comment:
Is it possible to use the `peer_response_time` defined in `STATE_DB.ASIC_TABLE` as a default value in case there is no pause quanta defined for the port's speed?
If we decided to change the default value in the future, we just need to update the default value in `STATE_DB`.
Otherwise, the `peer_response_time` in line 16 can be removed as it is not fetched from `STATE_DB.ASIC_TABLE` now.